README.md

```diff
@@ -49,6 +67,25 @@ Detailed instructions for running unit and end-to-end integration testing on fro
 - [Frontend](https://github.com/HHS/TANF-app/tree/main/tdrs-frontend)
 - [Backend](https://github.com/HHS/TANF-app/tree/main/tdrs-backend)
 
+## Troubleshooting
```

Comment:
@alexsoble @jtwillis92 I added this section to help with troubleshooting

Reply:
I love it! Thanks @carltonsmith! 😄